mne_bids/write.py

```diff
@@ -1432,6 +1432,7 @@ def write_raw_bids(
     event_id=None,
     event_metadata=None,
     extra_columns_descriptions=None,
+    electrodes_tsv_task=False,
```

Comment:
You placed the new parameter `electrodes_tsv_task=False` before the asterisk `*` - but I would actually move `electrodes_tsv_task=False`  _after it_. A bare asterisk in the function signature forces any parameters that are defined after it to be passed as keyword arguments when calling the function. And since this parameter is new, I think that it makes sense to force the user to pass it as a keyword-argument (`electrodes_tsv_task=False`), instead of as a positional one (`False`)